common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -233,15 +233,13 @@ public void set(String key, String value) {
     mProperties.put(key, value);
   }
 
-  public String get(String key, final String defaultValue) {
-    String raw = mProperties.getProperty(key, defaultValue);
-    String updated = lookup(raw);
-    LOG.debug("Get Tachyon property {} as {} with default {}", key, updated, defaultValue);
-    return updated;
-  }
-
   public String get(String key) {
-    return get(key, null);
+    if (!mProperties.containsKey(key)) {
```

Comment:
I agree. I will have a separate PR to refactor `TachyonConf` to use `Preconditions` for this `get` and make other getters calling this `get`